### PR TITLE
Integrated ad labels

### PR DESCRIPTION
**Pull request by walsh9**
Depends on https://github.com/nypublicradio/nypr-ads/pull/9
Depends on https://github.com/nypublicradio/nypr-design-system/pull/16

Use the yielded values from the updated nypr-ads to build the components that represent our different ad tag configurations. We can use the isEmpty status to control whether or not we show the label.  Also remove the standalone label from the sections template.

### CODE REVIEW

File: app/components/ad-tag-wide/template.hbs [hunk 1]
```diff
@@ -1,48 +1,49 @@
...
+      {{/if}}
+    </div>
   </div>
-</div>
-{{#if @breakMargins}}
-  <div class="space-filler"></div>
-{{/if}}
\ No newline at end of file
+  {{#if (and @breakMargins (not ad.isEmpty))}}
+    <div class="space-filler" style="height: {{ad.height}}px"></div>
```
Comment (noslouch) [inline app/components/ad-tag-wide/template.hbs]: ember will complain about this not being warpped in `htmlSafe`

Comment (walsh9) [reply app/components/ad-tag-wide/template.hbs]: looks like ember template lint doesn't even like `style=` 
https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/no-inline-styles.md

I get it, and don't really want to disable the rule, but still...
Seems like the alternatives here are go back to the list of classes (which is doable, the possible values are finite and known right now, but would need to be maintained) or set the height with js in the component.

or just put `{{! template-lint-disable no-inline-styles }}` in the template i guess

Comment (walsh9) [reply app/components/ad-tag-wide/template.hbs]: ah wait just read more closely on what's allowed in the allowDynamicStyles section. I can work with that

Comment (walsh9) [reply app/components/ad-tag-wide/template.hbs]: that option was actually just released 2 days ago https://github.com/ember-template-lint/ember-template-lint/releases/tag/v1.2.0

**Pull request merged**